be2-scala/src/main/scala/ch/epfl/pop/config/application.conf

```diff
@@ -17,12 +17,14 @@ akka {
 }
 
 #POP Server config default
+#Example of external address: wss://pop-demo.online, will default to ws://interface:port if empty
```

Comment:
Is that a safe behavior to have? The pop specification expect communication over TLS for security, so I don't believe this is a safe default value...

Reply:
This address is only relevant in case of reverse proxy usage. As far as I can tell the system doesn't support TLS natively, we can only rely on external solution. This default behavior when the field is empty is what we actually do today. Unless we add native support of TLS, I don't think we can have safe defaults, you are correct.